feature/system/gnmi/cliorigin/mixed_oc_cli_origin_support_test/mixed_oc_cli_origin_support_test.go

```diff
@@ -0,0 +1,146 @@
+/*
+Copyright 2022 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package gnmi_1_12_mixed_oc_cli_origin_support_test implements GNMI 1.12 from go/wbb:vendor-testplan
+package gnmi_1_12_mixed_oc_cli_origin_support_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/openconfig/ygot/ygot"
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	gpb "github.com/openconfig/gnmi/proto/gnmi"
+	"github.com/openconfig/ondatra"
+)
+
+func TestMain(m *testing.M) {
+	fptest.RunTests(m)
+}
+
+func interfaceDescriptionCLI(dp *ondatra.Port, desc string) string {
+	switch dp.Device().Vendor() {
+	case ondatra.ARISTA:
```

Comment:
I believe Cisco uses the same syntax as well.
```suggestion
	case ondatra.ARISTA, ondatra.CISCO:
```